doc/user/content/integrations/monitor/datadog.md

```diff
@@ -0,0 +1,165 @@
+     - DD_SITE=<DATADOG_REGION_ADDRESS>
+...
+```
+
+## Deploy the SQL exporter and Datadog agents
+
+In the current directory, run Docker to deploy the SQL exporter and the Datadog
+agents. The SQL exporter tool uses the connection string you created in a
+previous step to connect with your Materialize instance. 
+
+```shell
+$ docker compose up -d
+```
+
+The Prometheus SQL exporter container runs a service that executes queries against your
+database and then exports the metrics in a format that most monitoring services
+can consume.
+
+The Datadog container runs an agent with account information to communicate
+with the cloud monitoring service. The `docker-compose.yaml` file deploys both
+of these images and configures the Datadog agent with the OpenMetrics
+specification to allow it to parse the metrics returned by the SQL exporter.
+
+## Configure the Datadog dashboard
+
+Now that the containers are running, your Datadog agent will reach out to the
+Datadog endpoint. No metrics are being displayed yet, so let's configure the
+dashboard with some important information.
+
+In the `datadog` directory, copy the contents of the `dashboard.json` file.
+
+In your Datadog instance, navigate to the **Dashboards** section and select 
+**New Dashboard** in the upper right section. Name and create the new dashboard
+then find the gear icon in the upper right navigation. Select **Import dashboard
+JSON** and paste the contents of the `dashboard.json` file.
+
+Once you import, your dashboard will display information
+for your Materialize resources.
+
+## Helpful queries
+
+This dashboard returns a lot of useful information about your Materialize
+resources, but there are some more metrics in the system catalog that you may
+find useful.
+
+In your `config.yml` file, you can add additional metrics to the SQL exporter.
+To display specific information about a table, add  a new query to the
+configuration.
+
+```sql
+queries:
+- name: "total orders"
+   help: "Total Orders"
+   values:
+   - "count"
+   query:  |
+           SELECT count(*) FROM orders
+```
+
+To add more complex queries, you can use the template above and edit the query
+and values you expect the query to return.
+
+
+Most queries reference the `mz_internal` schema. For more information, visit the [`mz_internal` reference documentation](https://materialize.com/docs/sql/system-catalog/mz_internal/).
+
+Monitor operator speed across workers:
+
+```sql
+SELECT mdo.id, mdo.name, mse.elapsed_ns
+FROM mz_internal.mz_scheduling_elapsed AS mse,
+     mz_internal.mz_dataflow_operators AS mdo
+WHERE mse.id = mdo.id
+ORDER BY elapsed_ns DESC;
+```
+
+Monitor operator speed by worker:
+
+```sql
+SELECT mdo.id, mdo.name, mse.worker_id, mse.elapsed_ns
+FROM mz_internal.mz_scheduling_elapsed_per_worker AS mse,
+     mz_internal.mz_dataflow_operators AS mdo
+WHERE mse.id = mdo.id
+ORDER BY elapsed_ns DESC;
+```
+
+Monitor active `SUBSCRIBE` dataflows:
+
+
+```sql
+SELECT count(1) FROM (
```

Comment:
Just [a couple of hours ago](https://github.com/MaterializeInc/materialize/pull/20915#event-9978880001) this query was removed from the troubleshooting section in favor of using `mz_subscriptions`.

The updated query to monitor active `SUBSCRIBE` dataflows is as follows:

```sql
SELECT COUNT(1) FROM mz_internal.mz_subscriptions;
```